### PR TITLE
minor updates

### PR DESCRIPTION
**Pull request by ngarnier**

### CODE REVIEW

File: site/template.html [hunk 1]
```diff
@@ -13,10 +13,10 @@
   <script>try{Typekit.load({ async: true });}catch(e){}</script>
 
   <div class="d-f ai-c p-20 flow-h-20" style="margin-bottom: 120px">
-    <a href="/">
+    <a href="https://mjml.io">
       <img height="40" src="assets/logo-mjml.svg" />
     </a>
-    <a target="_blank" href="https://mjml.io">About</a>
+    <a target="_blank" href="https://github.com/mjmlio/mjml#introduction">About</a>
```
Comment (ngarnier) [inline site/template.html]: Added the link to the MJML website on the MJML logo which seemed more user friendly, so maybe About should link to something else? I linked it to the Readme, what do you think?

File: site/template.html [hunk 2]
```diff
@@ -147,7 +147,9 @@
   </div>
 
   <div class="footer z">
-    <img src="assets/logo-footer.png" style="opacity: 0.5" />
+    <a href="https://mjml.io">
```
Comment (ngarnier) [inline site/template.html]: Added a link to mjml.io on the footer logo

**Comment by meriadec** [comment]
Thx. deploying it.

File: site/template.html (hunk 1 above)
Comment (meriadec) [inline site/template.html]: total yes

**Pull request merged**